.commitlintrc.mjs

```diff
@@ -0,0 +1,48 @@
+import * as assert from "node:assert";
+
+let gimojis;
+try {
+    const res = await fetch(
+        "https://raw.githubusercontent.com/zeenix/gimoji/refs/heads/main/emojis.json",
+    );
+    assert.ok(res.ok);
+    const json = await res.json();
+    gimojis = json.gitmojis.map(({ emoji }) => emoji);
+} catch (cause) {
+    throw new Error("unable to fetch gimoji emojis.json", { cause });
+}
+
+// extra symbols from neither gitmoji nor gimoji
+const extras = ["🗑"];
+
+export default {
+    defaultIgnores: true,
+    helpUrl: "https://github.com/z-galaxy/zbus/blob/main/CONTRIBUTING.md",
+    ignores: [
+        (commit) =>
+            commit.includes(
+                "This PR was generated by [Mend Renovate](https://mend.io/renovate/).",
+            ),
+    ],
+    plugins: [
+        {
+            rules: {
+                "start-with-gitmoji": ({ header }) => {
+                    return [
+                        [gimojis, extras].some((collection) =>
+                            collection.some((symbol) =>
+                                header.startsWith(symbol),
+                            ),
+                        ),
+                        "Your subject should start with an appropriate Unicode emoji from https://gitmoji.dev/",
```

Comment:
We still point to gitmoji and variables/names also need to be updated.

Reply:
now points at `gimoji` GitHub repository